feat: move tasks to workers (#689)
callstack/haul: docs/CLI Commands.md

Path to the webpack haul config. Defaults to `haul.config.js`.

#### `--progress <string>`

Display bundle compilation progress with different verbosity levels. Accepts `none`, `minimal`, `compact`, `expanded` and `verbose`. Note that logging the compilation progress will increase build time. Defaults to `none` when `--dev` is set to `false`.

#### `--max-workers <number>`

> Number of workers used to transpile modules. Defaults to the number of the CPUs - 1. Default is limited to `7` on CI, as the number of CPUs happen to be not correct.

Reviewer: ```suggestion
Number of workers used to transpile modules. Defaults to the number of the CPUs - 1 on local machine, but is limited to `7` on CI, as the number of CPUs can happen to be incorrect.
```

#### `--port <number>`

Port on which the packager server is running. Defaults to `8081`.

#### `--progress <string>`

Display bundle compilation progress with different verbosity levels. Accepts `none`, `minimal`, `compact`, `expanded` and `verbose`. Note that logging the compilation progress will increase build time. Defaults to `none` when you are building in production mode.

#### `--max-workers <number>`

> Number of workers used to minify RAM bundle and transpile modules. Defaults to the number of the CPUs - 1. Default is limited to `7` on CI, as the number of CPUs happen to be not correct.

Reviewer: ```suggestion
Number of workers used to transpile modules. Defaults to the number of the CPUs - 1 on local machine, but is limited to `7` on CI, as the number of CPUs can happen to be incorrect.
```
